openbb_terminal/stocks/fundamental_analysis/fmp_view.py

```diff
@@ -525,3 +526,58 @@ def display_financial_statement_growth(
     else:
         logger.error("Could not get data")
         console.print("[red]Could not get data[/red]\n")
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_KEY_FINANCIALMODELINGPREP"])
+def display_filings(pages: int = 1, limit: int = 5, today: bool = False) -> None:
+    """Display recent forms submitted to the SEC
+
+    Parameters
+    ----------
+
+    pages: int = 1
+        The range of most-rececnt pages to get entries from (1000 per page, max 30 pages)
+    limit: int = 5
+        Limit the number of entries to display (default: 5)
+    today: bool = False
+        Show all from today
+    """
+    filings = fmp_model.get_filings(pages)
+    if today is True:
+        now: str = datetime.datetime.now().strftime("%Y-%m-%d")
+        iso_today: int = datetime.datetime.today().isoweekday()
+        if iso_today < 6 and not filings.empty:
+            filings = filings.filter(like=now, axis=0)
+            limit = 1000
+        else:
+            console.print(
+                "[red]No filings today, displaying the most recent submissions instead[/red]"
+            )
+    else:
```

Comment:
Because there is no code here, there is no need to have the today argument.  its a redundant argument

Reply:
How is it redundant? It checks if today == weekday, and if not, print that message and returns the default of most recent five.

Reply:
Thats my bad,  I was switching 546 and 547 in my head so that if today=False, iit was not doing anything.  The else: pass is not needed because if its False the block will not be evaluated.

Reply:
Gotcha. Removed `else: pass`